CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* Bump Cosmos-SDK to v0.45.10-pio-4 (from v0.45.9-pio-3) [PR 1202](https://github.com/provenance-io/provenance/pull/1202)
```

Comment:
Do we need both this line and the previous? seems like this update fully supersedes the one below it...